zebra-test/src/command.rs

```diff
@@ -306,7 +308,9 @@ impl<T> TestOutput<T> {
             return Ok(self);
         }
 
-        Err(eyre!("stdout of command is not equal the given string")).context_from(self)
+        Err(eyre!("stdout of command is not equal the given string"))
+            .context_from(self)
+            .with_section(|| format!("{:?}", s).header("Match String:"))
```

Comment:
:fire:

Reply:
Failures are a bit confusion without this change